.github/workflows/e2e-test-k8s.yml

```
@@ -34,35 +34,27 @@ jobs:
       matrix:
         cases_subset:
         - v2
-    runs-on: buildjet-2vcpu-ubuntu-2004
+    runs-on: self-hosted
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)